### PR TITLE
Fix #40: Added load tests

### LINKED ISSUE #40: Performance benchmarks
Depend on #37 #25

### PR DESCRIPTION
**Pull request by Fly-Style**
Using [PhantomJS](http://phantomjs.org/) and Node.js like a JS engine, begin work with load tests. 
This is really simple tests. First [simple load test](https://github.com/fomkin/korolev/blob/issue-40/korolev-phantomjs-tests/src/simple_test.js) only requests start page of [routingExample](https://github.com/fomkin/korolev/blob/issue-40/examples/routing/src/main/scala/RoutingExample.scala). 
[Second load test](https://github.com/fomkin/korolev/blob/issue-40/korolev-phantomjs-tests/src/simple_emulaiton.js) emulates user actions - it marks all todo`s without first as done. 
Tests runs with 500 requests as load.

### CODE REVIEW

File: korolev-phantomjs-tests/src/simple_test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
```
Comment (fomkin) [inline korolev-phantomjs-tests/src/simple_test.js]: What the difference between `simple_test.js` and `simple_emulation.js`?

File: korolev-phantomjs-tests/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by flystyle on 24.01.17.
+ */
+
+const gulp = require('gulp');
+const babel = require('gulp-babel');
```
Comment (fomkin) [inline korolev-phantomjs-tests/gulpfile.js]: Do you sure we really need babel and gulp here? I suppose it's redundant.

File: korolev-phantomjs-tests/dist/simple_emulaiton.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+for (var i = 0; i < requests; i++) {
+    var page = require('webpage').create();
+    page.onResourceReceived = onResourceReceived;
+    page.onResourceRequested = onResourceRequested;
+    page.open(url + "?i=" + i, function () {
+        page.evaluate(function () {
+            var posts = document.getElementsByClassName("checkbox");
+            for (var j = 1; j < posts.length; j++) {
+                console.log(posts[j].className);
+                posts[j].className = "checkbox checkbox__checked";
```
Comment (fomkin) [inline korolev-phantomjs-tests/dist/simple_emulaiton.js]: Well. It tests nothing. You just set class name. Korolev doesn't get it. Here you should emit click event with bubbling.

File: korolev-phantomjs-tests/src/simple_emulaiton.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    page.open(url + "?i=" + i, () => {
+        page.evaluate(() => {
+            let posts = document.getElementsByClassName("checkbox");
+            for (var j = 1; j < posts.length; j++) {
+                console.log(posts[j].className);
+                posts[j].className = "checkbox checkbox__checked";
+            }
+        });
+        const val = Date.now() - startTime;
+        setTimeout(() => {}, 1000);
```
Comment (fomkin) [inline korolev-phantomjs-tests/src/simple_emulaiton.js]: What is mean?

File: korolev-phantomjs-tests/src/simple_test.js (hunk 1 above)
Comment (Fly-Style) [inline korolev-phantomjs-tests/src/simple_test.js]: Second script emulates user behaviour. First only load root page.

File: korolev-phantomjs-tests/gulpfile.js (hunk 2 above)
Comment (Fly-Style) [inline korolev-phantomjs-tests/gulpfile.js]: PhantomJS doesn`t support ES6 which allows writing code faster. It`s cause I used babel and gulp for distribution of ES5 code from mine ES6.

Comment (fomkin) [inline korolev-phantomjs-tests/gulpfile.js]: Did you try this https://github.com/amir20/phantomjs-node ?

File: korolev-phantomjs-tests/src/simple_test.js (hunk 1 above)
Comment (fomkin) [inline korolev-phantomjs-tests/src/simple_test.js]: We need only second one.

File: korolev-phantomjs-tests/gulpfile.js (hunk 2 above)
Comment (Fly-Style) [inline korolev-phantomjs-tests/gulpfile.js]: I can try it, sure.

**Pull request closed without merging**